integration/src/test/resources/non-blocking-test-config.xml

```diff
@@ -156,7 +156,6 @@
     </flow>
 
     <flow name="transactionalTry" initialState="stopped">
-        <test:assert-intercepting responseSameTask="false" responseExpression="#[payload == 'Test Message']"/>
```

Comment:
we should not lose coverage for this scenario. Parameterize it to continue executing with the proactor PS